apps/angular/5-crud-application/src/app/app.component.ts

```diff
@@ -1,51 +1,29 @@
 import { CommonModule } from '@angular/common';
-import { HttpClient } from '@angular/common/http';
-import { Component, OnInit } from '@angular/core';
-import { randText } from '@ngneat/falso';
-
+import { Component, signal } from '@angular/core';
+import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
+import { ToDoComponent } from '../components/to-do/to-do.component';
+import { TodoDetails } from '../model/user';
+import { CrudSpecialService } from '../services/crud-special.service';
 @Component({
   standalone: true,
-  imports: [CommonModule],
+  imports: [CommonModule, MatProgressSpinnerModule, ToDoComponent],
   selector: 'app-root',
-  template: `
-    <div *ngFor="let todo of todos">
-      {{ todo.title }}
-      <button (click)="update(todo)">Update</button>
-    </div>
-  `,
+  templateUrl: './app.component.html',
   styles: [],
 })
-export class AppComponent implements OnInit {
-  todos!: any[];
+export class AppComponent {
+  public loading = true;
+
+  constructor(private todoService: CrudSpecialService) {}
```

Comment:
question: did you try the inject function ? Or do you prefer to stay with the constructor